### ISSUE TITLE
Reduce code duplication in Xbase tests

### ISSUE THREAD

**Issue by szarnekow** [eclipse/xtext-extras#809]
The AbstractXbaseEvaluationTests are unnecessarily duplicated in the code-base.

**Comment by cdietrich** [comment]
afaik there is also diffs betwen them

**Comment by szarnekow** [comment]
> afaik there is also diffs betwen them

Yes, that's true. I merged the changes into the base class and marked the test-cases, that are not supported by default, as ignored. It was already done like that in one of the dups.

**Comment by szarnekow** [comment]
Merged

**Comment by szarnekow** [comment]
There is more of it.

**Comment by cdietrich** [comment]
@szarnekow  any update

**Comment by cdietrich** [comment]
move to 2.30 for remainders
